### PR TITLE
MESSAGEix manuscript reference update #107

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR updates the references in the documentation, notice, etc. to the published version of the MESSAGEix manuscript and to the latest version of the MESSAGEix South Africa manuscript.

The changes are the equivalent updates to https://github.com/iiasa/ixmp/pull/107

### CODE REVIEW

**Comment by danielhuppmann** [comment]
@gidden left for an extended Christmas vacation, let's not bother him with that

**Comment by danielhuppmann** [comment]
same formatting issue as identified by @khaeru in https://github.com/iiasa/ixmp/pull/107

File: doc/source/api.rst [hunk 1]
```diff
@@ -1,9 +0,0 @@
-`ixmp` API Documentation
-------------------------
-
-.. include:: api/ixmp.rst
-
-`message_ix` API Documentation
-------------------------------
-
-.. include:: api/message_ix.rst
```
Comment (khaeru) [inline doc/source/api.rst]: Also made these changes in #138, but I can rebase that other PR after this one is merged.

File: doc/source/model.rst [hunk 2]
```diff
@@ -43,6 +43,8 @@ There are three methods to run the |MESSAGEix| model:
   The scenario name and other arguments can be passed as command line parameters, 
   e.g. :literal:`gams MESSAGE_run.gms --in="<data-file>" --out="<output-file>"`.
 
+.. _`tutorials`: tutorials.html
+
```
Comment (khaeru) [inline doc/source/model.rst]: Also touched in #138, but a rebase will handle.

**Comment by danielhuppmann** [comment]
merging as approved by @khaeru

**Pull request merged**